qabelbox/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
         add code to automaticly generate version name and code,
         on base of timestamp or change it manual*/
 
-        versionCode System.getenv("BUILD_NUMBER") as Integer ?: 1
+        versionCode System.getenv("BUILD_NUMBER") as Integer ?: 3
         versionName "0.2." + (System.getenv("BUILD_NUMBER") as Integer ?: 3)
```

Comment:
According to the version scheme currently in use now this should result in `0.2.4`.